source/grid/grid_tools_nontemplates.cc

```diff
@@ -57,8 +57,8 @@ namespace GridTools
                              all_vertices[vertex_indices[1]](1),
                              all_vertices[vertex_indices[2]](1)};
 
-        return 0.5 * std::abs((x[0] - x[2]) * (y[1] - y[0]) -
-                              (x[0] - x[1]) * (y[2] - y[0]));
+        return 0.5 *
```

Comment:
I am not sure why abs() was used here before. @peterrum , do you know why?

Reply:
I guess the person who wrote this (probably me) did not think of negative determinants^^